### PR TITLE
RDC Sprint3 Staging

### LINKED ISSUE #86: Customize Title display on Dataset entities
If dc.title.dataset and dc.title.project are both available, render like:

<dc.title.dataset>: Supplement to <dc.title.project>.

Do this on the list view and the simple item view.

If only one is available, fall back on it as the single title and display normally.

### PR DESCRIPTION
**Pull request by jsavell**
## References
Resolves #86 

## Description
Updates the Title display in Single Item and Browse views for Dataset entities to show dc.title.project, dc.title.dataset, or both.

### CODE REVIEW

File: src/app/core/breadcrumbs/dso-name.service.ts [hunk 1]
```diff
@@ -95,7 +96,7 @@ export class DSONameService {
     const types = dso.getRenderTypes();
     const entityType = types
       .filter((type) => typeof type === 'string')
-      .find((type: string) => (['Person', 'OrgUnit']).includes(type)) as string;
+      .find((type: string) => (['Person', 'Dataset']).includes(type)) as string;
```
Comment (wwelling) [inline src/app/core/breadcrumbs/dso-name.service.ts:99]: We need comments around this customization.

File: src/app/core/breadcrumbs/dso-name.service.ts [hunk 2]
```diff
@@ -105,10 +106,25 @@ export class DSONameService {
         return familyName || givenName;
       }
       return `${familyName}, ${givenName}`;
-    } else if (entityType === 'OrgUnit') {
-      return this.firstMetadataValue(object, dso, 'organization.legalName') || this.translateService.instant('dso.name.untitled');
+      // TAMU Customization - swithched the entitiy to look for DATASET in stead of orgUnit
```
Comment (wwelling) [inline src/app/core/breadcrumbs/dso-name.service.ts:109]: Please fix typos.

File: src/themes/rdc/app/item-page/simple/item-types/untyped-item/untyped-item.component.html [hunk 3]
```diff
@@ -14,6 +14,7 @@
   </ds-themed-item-page-title-field>
   <ds-dso-edit-menu></ds-dso-edit-menu>
 </div>
+
```
Comment (wwelling) [inline src/themes/rdc/app/item-page/simple/item-types/untyped-item/untyped-item.component.html:17]: Let's try to keep the changes to files in the themes that are duplicate of core code to a minimal to simplify checking to synchronize on next update.

File: src/app/item-page/simple/field-components/specific-field/title/item-page-title-field.component.html [hunk 4]
```diff
@@ -1,6 +1,30 @@
-<h1 class="item-page-title-field">
+<!-- // Core code block- Left intact on purpose -->
```
Comment (wwelling) [inline src/app/item-page/simple/field-components/specific-field/title/item-page-title-field.component.html:1]: Can we please use consistency with customization comments.

**Review by wwelling — CHANGES_REQUESTED** [review]

File: src/app/item-page/simple/field-components/specific-field/title/item-page-title-field.component.html (hunk 4 above)
Comment (qtamu) [reply src/app/item-page/simple/field-components/specific-field/title/item-page-title-field.component.html:1]: Not a TAMU customization

Comment (wwelling) [reply src/app/item-page/simple/field-components/specific-field/title/item-page-title-field.component.html:1]: image

This is a core code customization removing the core template for this component and replacing it with our own.

Are we not able to override this component in the themes?

**Review by wwelling — APPROVED** [review]
I am approving this for now. But I think it warrants double checking if we can move more customizations into the themes.

**Pull request merged**